shared/utils/escape.ts

```diff
@@ -0,0 +1,4 @@
+// source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
+export const escapeRegExp = (text: string) => {
```

Comment:
This utility exists in lodash, please use that one

Reply:
Thanks for the tip :+1:

Reply:
Fixed!